packages/renderer/src/lib/preferences/PreferencesRenderingItem.svelte

```diff
@@ -6,23 +6,21 @@ import type { IConfigurationPropertyRecordedSchema } from '../../../../main/src/
 import Markdown from '../markdown/Markdown.svelte';
 import PreferencesRenderingItemFormat from './PreferencesRenderingItemFormat.svelte';
 
-export let record: IConfigurationPropertyRecordedSchema;
+interface Props {
+  record: IConfigurationPropertyRecordedSchema;
+}
 
-let recordUI: {
-  title: string;
-  breadCrumb: string;
-  description?: string;
-  markdownDescription?: string;
-  original: IConfigurationPropertyRecordedSchema;
-};
+const { record }: Props = $props();
+let showResetButton = $state(false);
+let resetToDefault = $state(false);
 
 // add space from camel case and upper case on the first letter
 function startCase(str: string): string {
   return str.replace(/([A-Z])/g, ' $1').replace(/^./, str => {
     return str.toUpperCase();
   });
 }
-function update(record: IConfigurationPropertyRecordedSchema) {
+const recordUI = $derived.by(() => {
```

Comment:
Same remark as https://github.com/podman-desktop/podman-desktop/pull/10493#discussion_r1900658398 `const` or `let` ?

Reply:
I would say it's six of one, half a dozen of the other as it'll be intercepted by the svelte compiler anyway